src/components/MetricValue.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const MetricValue = props => (
```

Comment:
this is not what i meant :/
what if you add a third metric?!

this is what you want
```
const MetricValue = ({ label, value }) => (
  <div>
    <label>{label}</label>
    <span className="metric-value">{value}</span>
  </div>
);
```
also note prop deconctruction: `({ label, value })`

Reply:
0c2a405 

that's how you would answer change requests